binder/views.py

```diff
@@ -1510,6 +1522,50 @@ def _apply_q_with_possible_annotations(self, queryset, q, annotations):
 		return queryset.filter(q)
 
 
+	def _after_expr(self, request, after):
```

Comment:
Can you add a small description of what this does? From the name alone, it is not very clear what it is supposed to do, and how it should be used